docs/docs/index.md

```diff
@@ -112,8 +112,13 @@ print len(q)
 queued_job_ids = q.job_ids # Gets a list of job IDs from the queue
 queued_jobs = q.jobs # Gets a list of enqueued job instances
 job = q.fetch_job('my_id') # Returns job having ID "my_id"
+
+# Deleting the queue
+q.delete(true) # Passing true here additionally removes all the associated messages on the queue
```

Comment:
Could you please change this to `q.delete(delete_jobs=True)`? It'd be much clearer what the argument is supposed to do if we show the keyword argument.